projects/vanilla/src/app/app.module.ts

```diff
@@ -84,9 +101,9 @@ import { SearchFormComponent } from "@sinequa/components/search-form";
 
 // Initialization of @sinequa/core
 export const startConfig: StartConfig = {
-    // app: "training",
+    app: "training",
```

Comment:
The app name and SAML provider should remain commented, as this app is included in the build of the product.